packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -128,9 +128,9 @@ class CallContext {
     const oldPage = this.state.page;
     const newPage = getCallCompositePage(call, latestEndedCall);
     if (!IsCallEndedPage(oldPage) && IsCallEndedPage(newPage)) {
-      this.emitter.emit('callEnded', {
-        callId: this.callId
-      });
+      this.emitter.emit('callEnded', { callId: this.callId });
+      // Reset the callId to undefined as the call has ended.
+      this.setCurrentCallId(undefined);
```

Comment:
Do you no longer need to resync state from stateful layer after the `callId` is set (i.e., what happened to the old line 388)?